src/constants.js

```diff
@@ -7,6 +7,7 @@ export const ASYNC_RENDER = 3;
 
 
 export const ATTR_KEY = '__preactattr_';
+export const ELT_KEY_PREFIX = 'preact-';
```

Comment:
Do you need the prefix?

Reply:
It was a way to avoid ambiguity if the user explicitly set numeric keys for some of the elements (but not all). I don't know if we should really optimize for this use-case though